assignments/chess-board/code.js

```diff
@@ -38,11 +38,21 @@ let board =
 const row = (y) => Math.floor(y / (height / 8))
 const column = (x) => Math.floor(x / (width / 8))
 
-const select = (x, y) => {
-  drawText(board[row(y)][column(x)], width / 8 * column(x), height / 8 * (row(y) + 1), 'red', 45)
+let aSelect = []
```

Comment:
So, as we were discussing today before your computer got haunted, I'd change this like this:

```suggestion
let selectedPiece = null
```